cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationConfiguration.kt

```diff
@@ -372,6 +385,26 @@ private constructor(
             return this
         }
 
+        /** Produces the [Pass] with the given name. */
```

Comment:
Sounds like a private or internal function

Reply:
Resolved in 979866b61.